seqr/views/utils/variant_utils.py

```diff
@@ -472,8 +472,9 @@ def _set_response_gene_scores(response, family_genes, gene_ids):
 
 def _add_sample_count_stats(response):
     sample_counts = Sample.objects.filter(
```

Comment:
when I did the initial global AF deprecation I only did it for SNV_INDEL - this updates the UI to work with any dataset type